feat(DEVSITE-114): add logic for "keywords" processing (#4)
AdobeDocs/frontmatters: README.md

2. Answer the brief on-screen prompts. 
   
More details to explain valid answers to the prompts soon!

> ### "Parameters" way

Reviewer: Add description and example for the valid source path structure: You must use `./` to prefix your path from your current location. And don't end your path the a forward slash `/'.

Correct source path:
```
[~/Projects/_AdobeIO_Sites]$ npx https://github.com/AdobeDocs/frontmatters.git --ignore-existing -r uxp-xd -s ./uxp-xd/src/pages
```

Incorrect source path:
```
[~/Projects/_AdobeIO_Sites]$ npx https://github.com/AdobeDocs/frontmatters.git --ignore-existing -r uxp-xd -s /uxp-xd/src/pages/
```

```bash
    npx https://github.com/AdobeDocs/frontmatters.git --ignore-existing -r [repository] -s [source]
```

> Where is:

Reviewer: ```suggestion
Where:
```

> - `repository` `string` `required` - the repo which you want to frontmatter;

Reviewer: ```suggestion
- `[repository]` (required, string) is the name of the GitHub repository where the markdown files for your documentation are stored.
```

> - `source` `string` `required` - the path to the folder for the repo's markdown files.

Reviewer: ```suggestion
- `[source]` (required, string) is the path to your local markdown files from your GitHub repo. This is a relative path from your current location in the terminal. The path must be prefixed wtih `./` and must not end in a forward slash `/`. See examples.
```
